internal/pkg/rules/rules.go

```diff
@@ -189,6 +193,11 @@ func syncUpdates(ctx context.Context, conf *config.Config, configDir, token, upd
 	return currRulesPath, nil
 }
 
+func isKrewPluginEnabled() bool {
+	log.Debug().Bool("enabled", len(krewPluginEnabled) > 0).Msg("Krew plugin")
+	return len(krewPluginEnabled) > 0
```

Comment:
where is `krewPluginEnabled` assigned anything?

Reply:
It will be in the preq-staging repo Makefile via LD FLAGS

Reply:
https://github.com/prequel-dev/preq-staging/pull/6/files

Reply:
ahha! maybe add comment about this?